src/api/api.ts

```diff
@@ -70,9 +93,13 @@ async function requestAPI<RequestType extends RequestDto, ResponseType extends R
       },
     );
 
+    if (response.status === StatusCodes.NO_CONTENT) {
+      return { error: 'no_error', code: response.status, body: null as ResponseType };
+    }
```

Comment:
Est ce que ce serait pas plus logique de renvoyer un body vide plutôt que `null` ? Puisque bon ça reste une requête post à priori donc elle a tjrs un body. Qui peut être vide, certes (comme l'indique le code http) mais qui est là quand même
```suggestion
    if (response.status === StatusCodes.NO_CONTENT) {
      return { error: 'no_error', code: response.status, body: {} as ResponseType };
    }
```

Reply:
je pense pas, parce que [] ça reste aussi un objet JSON valide, donc pourquoi ça serait {} ?
et si c'est en NO_CONTENT, c'est qu'il n'y a rien, {} c'est déjà quelque chose. A mon avis ça a plus de sens de mettre `null`, qui veut vraiment dire "rien"